readthedocs/oauth/services/github.py

```diff
@@ -446,8 +446,8 @@ def send_build_status(self, build, commit, status):
         project = build.project
         owner, repo = build_utils.get_github_username_repo(url=project.repo)
 
-        # select the correct state and description.
-        github_build_state = SELECT_BUILD_STATUS[status]["github"]
+        # select the correct status and description.
+        github_build_status = SELECT_BUILD_STATUS[status]["github"]
```

Comment:
I'm not sure that the rename of the `state` to `status` helps here. We call it `status` in some places and GitHub calls it `state`. Probably makes sense to keep consistency in our code, tho 👍🏼

Reply:
I have changed this because this is definitely about status. GitHub also calls it status AFAICT? https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28

Reply:
We are sending `state` in the data we send to them.

Reply:
No, we're sending the status :) I know that it's meaningless to talk about a real difference between these two words, but I think we only use "state" to refer to "build state" and everything else is "status".

I think that clearly when it's called `SELECT_BUILD_STATUS[status]["github"]`, it is correct to call the variable `github_build_status` and it did make me _less_ confused after I renamed it :)

Reply:
We send the `state` to GitHub here https://github.com/readthedocs/readthedocs.org/pull/10369/files/1b1d9fe39fa3de359fb13e9189c3f887b53dab7d#diff-99df3e1a60131d41823464e1ca35dcf068573dffb2ea4ce12dc68968c0ec71beR464

Reply:
Ah okay, my brain thinks about "GitHub commit status" - but I can see they called a field in the API "state", perhaps it was to avoid mixing it up with HTTP status. I think we should try to simplify our terminology rule to saying that we have "build state" for the Build objects internal naming of states - and then we have status for everything else. And then imagining a better future, we do a refactor to stop calling it "Build state" and call that "Build status" as well, so everything is "status" and we don't have "state" suddenly pop up and cause confusion :)